validator/src/cli.rs

```diff
@@ -1989,6 +1951,27 @@ fn deprecated_arguments() -> Vec<DeprecatedArg> {
                 Ok(())
             }
         }));
+    add_arg!(Arg::with_name("accountsdb_repl_bind_address")
```

Comment:
Sorry, I'm not too familiar with DeprecatedArgs yet. These don't get re-exposed via `--help` or anything, do they?

Reply:
Good question, I had to sanity check this for myself too. These arguments have the hidden setting applied:
https://github.com/anza-xyz/agave/blob/158c4e05d5e60c5f5b3dc8e47f5f60a90beca6e1/validator/src/cli.rs#L2089-L2098

Reply:
And semi-related - we will emit a `warn!()` log if any of the deprecated ones are used (this is called right near the top of `main()`):
https://github.com/anza-xyz/agave/blob/158c4e05d5e60c5f5b3dc8e47f5f60a90beca6e1/validator/src/cli.rs#L2100-L2122

Reply:
:shipit: